library/src/components/Histogram.vue

```diff
@@ -0,0 +1,103 @@
+<template lang="pug">
+  ag-charts-vue(:options="fullOptions")
+</template>
+
+
+<script>
+import { AgChartsVue } from "ag-charts-vue"
+export default {
```

Comment:
I think you need to define the component name like 
`name: 'nio-histogram'`

Otherwise once the library is built the export will reference a component by the name string that doesn't exist